block1_bugrace_intro/README.md

```diff
@@ -2,7 +2,7 @@
 
 Nu ska du få lära känna Scratch och samtidigt skapa ett roligt spel som är enkelt även om du aldrig har jobbat med Scratch tidigare. Målet är att få en skalbagge att springa runt en bana och försöka undvika att hamna utanför banan genom att styra med vänster- och högerpil på tangentbordet.
 Ett exempel på det färdiga spelet finns på: [https://scratch.mit.edu/projects/27697024/](https://scratch.mit.edu/projects/27697024/)
-<a href="www.dn.se" target="_blank">test</a>
+<br><a href="http://www.dn.se" target="_blank">test</a>
```

Comment:
Föreslår att markdown ska va för att enkelt klassificera innehåll. Dvs rubriker, bilder, länkar, osv. Styling och formattering är helt utanför det. Vi kan ju vilja generera PDF'er eller annat, så inline HTML eller annat borde undvikas helt.

Helst borde kodboken plocka upp någon "hint" om att vi vill öppna i nytt fönster, utan att vi lägger in icke-markdown. Här är lite förslag: http://stackoverflow.com/questions/4425198/markdown-target-blank